javatests/arcs/core/entity/HandleManagerTestBase.kt

```diff
@@ -713,7 +713,7 @@ open class HandleManagerTestBase {
         storageKey: StorageKey = singletonKey,
         name: String = "singletonWriteHandle",
         ttl: Ttl = Ttl.Infinite
-    ) = readHandleManager.createHandle(
+    ) = createHandle(
```

Comment:
This was resulting in the `Different*` tests not actually testing two different handle managers.